engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -161,11 +163,11 @@ public WorldRendererImpl(Context context, GLBufferPool bufferPool) {
         this.backdropProvider = context.get(BackdropProvider.class);
         this.renderingConfig = context.get(Config.class).getRendering();
         this.shaderManager = context.get(ShaderManager.class);
+        vrProvider = OpenVRProvider.getInstance();
         if (renderingConfig.isVrSupport()) {
-            this.vrProvider = new OpenVRProvider();
             context.put(OpenVRProvider.class, vrProvider);
-            if (this.vrProvider.init()) {
-                playerCamera = new OpenVRStereoCamera(this.vrProvider);
+            if (vrProvider.init()) {
+                playerCamera = new OpenVRStereoCamera(vrProvider);
                 currentRenderingStage = RenderingStage.LEFT_EYE;
             } else {
                 playerCamera = new PerspectiveCamera(renderingConfig.getCameraSettings());
```

Comment:
There's a comment in there now, but it seems to not have gotten picked up by Github since it's one line above this one.